core/micrometer/src/test/java/MicrometerExtensionIntegrationTest.java

```diff
@@ -0,0 +1,63 @@
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.eclipse.dataspaceconnector.azure.testfixtures.annotations.MicrometerIntegrationTest;
+import org.eclipse.dataspaceconnector.common.annotations.IntegrationTest;
+import org.eclipse.dataspaceconnector.junit.launcher.EdcExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.eclipse.dataspaceconnector.common.testfixtures.TestUtils.getFreePort;
+
+@IntegrationTest
+@MicrometerIntegrationTest
+@ExtendWith(EdcExtension.class)
+public class MicrometerExtensionIntegrationTest {
+    static final int CONNECTOR_PORT = getFreePort();
+    static final String CONNECTOR_URL = String.format("http://localhost:%s", CONNECTOR_PORT);
+    static final String HEALTH_ENDPOINT = String.format("%s/api/check/health", CONNECTOR_URL);
+    static final String METRICS_ENDPOINT = "http://localhost:9464/metrics";
+    static final String[] METRIC_PREFIXES = new String[]{
```

Comment:
```suggestion
    static final String[] METRIC_PREFIXES = new String[] {
```

reformat class

Reply:
Can you elaborate what you mean by reformat class?

Reply:
Just make sure you run IntelliJ reformatter and optimise imports, you can then resolve this